### ISSUE TITLE
No speech feedback whn downloading/installing language updates

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31741]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

With NVDA running go to:
Edit, Preferences and on the general Tab arrow down to the check for language updates button. Press enter.
If a dialogue appears saying there is an update when you press the download button nothing seems to happen.  The screen reader gives no verbal feedback to say it is downloading or installing the update or when it is complete.
The user is left thinking nothing is happening, but navigation does seem to freeze for a few seconds.
I am still unsure if the button worked and gave no speech or if it didn't activate as when I had someone sighted come and click the button with the mouse they said they saw some text that said downloading and then the window disappeared same as it had for me previously.  So probably just lack of speech and I had done it, but maybe someone could test the download button is fully keyboard accessible.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
